omnigan/losses.py

```diff
@@ -185,3 +185,73 @@ class L1Loss(MSELoss):
     def __init__(self):
         super().__init__()
         self.loss = torch.nn.L1Loss()
+
+
+def set_losses(opts, verbose):
```

Comment:
minor language issue but usually: setters modify an object in-place, getters return a value without modifying the object. It used to be a setter, now that it's not a Trainer method anymore it's a getter so name should be `get_`

Do you agree?

Reply:
totally!